src/components/UI/Charts/Sunburst/InfoPanelNavTree.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/prop-types */
+import React, { FunctionComponent } from 'react';
+import './sunburst.css';
```

Comment:
I know it's not in our guideline, but what do you think if we put all css import at the end of 'js' imports?